keyboards/sauce/mild/readme.md

```diff
@@ -0,0 +1,17 @@
+# Sauce Mild
+
+Mild is a basic TKL keyboard
+
+* Keyboard Maintainer: [Sauce](https://github.com/andyywz)
+* Hardware Supported: Mild pcb
+* Hardware Availability: -
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Done!